optimum/commands/onnxruntime/quantize.py

```diff
@@ -92,7 +92,10 @@ def run(self):
         elif self.args.avx512_vnni:
             qconfig = AutoQuantizationConfig.avx512_vnni(is_static=False, per_channel=self.args.per_channel)
         elif self.args.tensorrt:
-            qconfig = AutoQuantizationConfig.tensorrt(per_channel=self.args.per_channel)
+            raise ValueError(
+                "TensorRT quantization relies on static quantization that requires calibration, which is currently not supported through optimum-cli. Please adapt Optimum static quantization examples to run static quantization for TensorRT: https://github.com/huggingface/optimum/tree/main/examples/onnxruntime/quantization"
+            )
+            # qconfig = AutoQuantizationConfig.tensorrt(per_channel=self.args.per_channel)
```

Comment:
Although that is true, it could be implemented easily with the `TaskProcessor`s.
Also there is a comment to remove.

Reply:
Leaving the comment was intentional, but I'll remove it.